flycheck.el

```diff
@@ -1598,13 +1593,10 @@ are mandatory.
      If given this syntax checker is only used in buffers whose
```

Comment:
I think this should change, since the mode should always be given (I understand that the PR is still in flux :)